WarriorOfLeporia/WarriorOfLeporia/GameScene.swift

```diff
@@ -0,0 +1,225 @@
+//
+//  GameScene.swift
+//  WarriorOfLeporia
+//
+//  Created by Kirupa Nijastan on 2/28/18.
+//  Copyright © 2018 Kirupa Nijastan. All rights reserved.
+//
+
+import SpriteKit
+import GameplayKit
+
+struct PhysicsMasks {
+    static let None: UInt32 = 0 // 0
+    static let Player: UInt32 = 0b1 // 1
+    static let Enemy: UInt32 = 0b10 // 2
+    static let Bullet: UInt32 = 0b100 // 4
+    static let LevelBounds: UInt32 = 0b1000 // 8
+}
+
+
+class GameScene: SKScene, SKPhysicsContactDelegate {
+//    private var brabbit = SKSpriteNode()
+//    private var brabbitMoveFrames: [SKTexture] = []
+    var gameOver = false
+    var lastUpdateTime : TimeInterval = 0
+    
+    var leftButton: ButtonNode!
```

Comment:
Doing this works but is generally discouraged. If you use a bang "!" and the instance is null you'll have a null pointer exception.